.github/infrastructure/terraform/conformance/pubsub/aws/snssqs/snssqs.tf

```diff
@@ -0,0 +1,99 @@
+terraform {
+  required_version = ">=0.13"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.0"
+    }
+  }
+}
+
+provider "aws" {
+  region = "us-east-1"
+  default_tags {
+    tags = {
+      Purpose  = "AutomatedTesting"
+    }
```

Comment:
Could we add the CreationDate here too? So we could GC anything that is 2d old with no risk of impacting running tests.

Reply:
I thought we tear down everything immediately after the test run anyway?

Reply:
Ideally, yes. But if the `Terraform Destroy` step fails or isnt reached for whatever reason, there will be left over resources lingering around.

By tagging them we can identify those resources more easily from, say, any resource in that account that needs to be long lived. And it also allows us to limit the scope of the permissions required for the garbage collector.

Reply:
I think we should do that but it can be a follow-up PR